docs/csharp/toc.yml

```diff
@@ -1611,6 +1611,9 @@
     - name: Member access operators and expressions
       href: language-reference/operators/member-access-operators.md
       displayName: "., [], ?., ?[], (), indexer, null-conditional, Elvis, invocation, ^, index from end, hat, .., range"
+    - name: switch expressions
```

Comment:
Is it fine to move the TOC entry for the `switch` expression lower: just under `stackalloc`? The entries are sorted alphabetically, but only inside each of the two large groups: Overview articles and individual operator/expression articles are sorted separately.

Reply:
Good idea.